accounts/abi/event.go

```diff
@@ -69,6 +69,8 @@ func (e Event) tupleUnpack(v interface{}, output []byte) error {
 	for i := 0; i < len(e.Inputs); i++ {
 		input := e.Inputs[i]
 		if input.Indexed {
+			// indexed inputs are not available in log output
+			j--
```

Comment:
This seems fairly elegant. Works for me.

Reply:
Second thought, I'm not sure this plays nicely with a static array element insofar as indexing is concerned.

Reply:
thanks for review, i added required test, but i think that i spotted another issue with array types. please see my comment below